tests/extension/req-abort-custom-json-template.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Abort request as a result of rinit, with a custom template
+--INI--
+datadog.appsec.enabled=1
+--ENV--
+DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON=tests/extension/templates/response.json
```

Comment:
Do this work with absolute paths?

Reply:
Yes, but then it'll fail everywhere because the paths locally are different than the paths in CI.